ui/src/containers/Personas/PersonaManage/MergeForm/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { compose, withProps } from 'recompose';
+import classNames from 'classnames';
+import { withModel } from 'ui/utils/hocs';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import IdentifiersViewer from '../IdentifiersViewer';
+import AttributesViewer from '../AttributesViewer';
+import PersonaAutoComplete from '../PersonaAutoComplete';
+import MergeButton from './MergeButton';
+import styles from './styles.css';
+
+const enhance = compose(
+  withProps({ schema: 'persona' }),
+  withModel,
+  withStyles(styles)
+);
+
+const renderTarget = ({ hasTarget, mergeTargetId }) => {
+  if (!hasTarget) {
+    return <noscript />;
+  }
+
+  return (
+    <div>
+      <div className={styles.section}>
+        <h4>Identifiers</h4>
+        <IdentifiersViewer personaId={mergeTargetId} />
+      </div>
+      <div className={styles.section}>
+        <h4>Attributes</h4>
```

Comment:
?: `Attributes can be used to filter people in queries. A person can have multiple attributes with the same name.`

Reply:
We'll put this in the help centre and revisit if necessary. Discussed with @Ian247